### PR TITLE
Add Viper4android

### PR DESCRIPTION
**Pull request by StarkDroid**
With this Viper4Android should be working! :) although its untested so far, i'll get my next build with the new change.

### CODE REVIEW

**Comment by owlseye** [comment]
We will readd as soon as you tested. I planned on doing it (I added some fixes which should make it work) and forgot it.

**Comment by StarkDroid** [comment]
Okay so i've finalised the changes! Viper is now working and ready to be merged :)

File: prebuilt/common/su.d/50viper.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#! /system/bin/sh
+/system/xbin/supolicy --live "allow mediaserver mediaserver_tmpfs:file { read write execute };"
```
Comment (akhilnarang) [inline prebuilt/common/su.d/50viper.sh:2]: Why not add this one in external/sepolicy instead?
Just saying

Comment (owlseye) [inline prebuilt/common/su.d/50viper.sh:2]: I think I picked the commit on mm6.0 branch. Will add it on 6.0.1 and remove this.

**Comment by StarkDroid** [comment]
@akhilnarang you are the experienced one!! :) do what is right then.

File: prebuilt/common/su.d/50viper.sh (hunk 1 above)
Comment (akhilnarang) [inline prebuilt/common/su.d/50viper.sh:2]: OK

**Pull request merged**